korangar/src/loaders/script/mod.rs

```diff
@@ -131,7 +131,7 @@ end
         let globals = self.state.globals();
         let lookup_name = match is_identified {
             true => "identifiedResourceName",
-            false => "undidentifiedResourceName",
+            false => "unidentifiedResourceName",
```

Comment:
Did you verify that the current name is wrong? Because a lot of names in Ragnarok Online are incorrect/misspelled

Reply:
The game crashed and I fixed it.